tools/stimulus/functions-R.yaml

```diff
@@ -11,7 +11,10 @@
 #######################################
 
 igraph_empty:
-    IGNORE: RR
+
+# Not needed in R, we use igraph_emtpy() instead
+igraph_empty_attrs:
+    IGNORE: RR, RC
```

Comment:
Thanks. Why has the order changed now? I remember aligning this file with `functions.yaml` from the C core, I'd like both to be in the same order. No need to be religious about it, but let's keep the drift here to a minimum moving forward.

Reply:
I changed the order to be able to see both related functions next to it. If you're bothered by it, I'll change it in the C core too.

Reply:
Thanks. Changing in the C core would be great, if it's idiomatic there.

Reply:
Done.